evelink/parsing/planetary_interactions.py

```diff
@@ -0,0 +1,111 @@
+from evelink import api
+
+def parse_planetary_colonies(api_results):
+    result = {}
+    for rowset in api_results.findall('rowset'):
+        if rowset is None:
```

Comment:
I think I was worried at what happens if you have no colonies? But doesn't seem to break without it.